doc/templates/deprecated_class_with_call.rst

```diff
@@ -0,0 +1,24 @@
+:mod:`{{module}}`.{{objname}}
```

Comment:
It might be useful in the future is the idea, I think. Or he wasn't sure whether we deprecated any of the classes that require these templates